packages/eslint-plugin/docs/rules/sort-ngmodule-arrays.md

```diff
@@ -0,0 +1,33 @@
+# Enforces sorting of array identifiers for NgModule Properties (`sort-ngmodule-arrays`)
+
+This rule enforces that NgModule property arrays have a sorted list of identifiers. Objects such as provider definitions, call expressions and computed members will be ignored for sorting purposes. Sorting is based on simple string localeCompare.
```

Comment:
From @JamesHenry in https://github.com/angular-eslint/angular-eslint/pull/386#discussion_r603424611:

```suggestion
This rule enforces that NgModule metadata arrays have a sorted list of identifiers. Objects such as provider definitions, call expressions and computed members will be ignored for sorting purposes. Sorting is based on simple string localeCompare.
```